mitmproxy/contentviews/img/metadata.py

```diff
@@ -0,0 +1,30 @@
+from . import png
+
+class Metadata:
+    def __init__():
+        pass
+
+    def get_png_metadata():
```

Comment:
Multiple top level function for multiple image types? Or everything in just one function?